### PR TITLE
Please review; Update contributing guide

### PR DESCRIPTION
**Pull request by laurmurclar**
The contributing guidelines linked in the README were irrelevant. I removed the section and added the correct link to a more conventional CONTRIBUTING.md file. 

This will increase the community rating on the Insights page too.
screen shot 2017-11-04 at 12 56 36

### CODE REVIEW

**Comment by matthewmorgan** [comment]
@laurmurclar thanks, but I'm slightly unclear on what this change is going to do.  I agree that the docs linked to previously weren't right, and that your change is an improvement, but I think that contributors also need to see the information here: https://github.com/exercism/ecmascript/blob/master/README.md

The README contains information on how to run the test suites before making a PR.

@exercism/ecmascripts I think I missed the memo on these structural changes.  How do we ensure the contributors still see this README?

**Comment by laurmurclar** [comment]
@matthewmorgan I can add back the section to the README, but with 

>## Contributing Guide
>For an in-depth discussion of how exercism language tracks and exercises work, please see [CONTRIBUTING.md](https://github.com/exercism/ecmascript/blob/master/CONTRIBUTING.md)

instead if you prefer? 😊 That way, we still only have to ensure the link is up to date in one place, but don't remove the context.

This wasn't a decision made by the maintainers, I just thought it would be best to use the Github -wide standard. It makes it less confusing for new contributors, since the information is in the place they expect it.

**Comment by rchavarria** [comment]
I agree with @matthewmorgan. I also like having a link to the contributing guide in the main `README`. And I also agree with @laurmurclar, I like the idea of having a contributin guide in the track.

Sometimes, when I see @matthewmorgan suggest to contributors about squashing commits, or review canonical specs, I wonder if we should have a place (FAQ page?, contributing guide?) where we contributos could see detailed instructions about those questions. With the contributin guide inside the track, that would be the perfect place.

In summary, the `README` would link to the contributing guide. The contributing guide would link to a more general contributing guide. Then, we can expand this track contributing guide if we want. How does it sound?

**Comment by laurmurclar** [comment]
@rchavarria sounds perfect! I will make these changes later this afternoon 😄

**Comment by matthewmorgan** [comment]
Awesome!  @rchavarria look OK to you?

@laurmurclar can you please squash the commits to one?  Let me know if you need help with that.

**Comment by rchavarria** [comment]
Ok, let's move on this PR. It's not a big deal to have several commits instead of just one

**Comment by laurmurclar** [comment]
Sorry, completely forgot about this. For the future, I think there's a "squash and merge" option on the dropdown beside the merge button. 

image

**Comment by matthewmorgan** [comment]
@laurmurclar we know about the squash button! ;)  Not all of our contributors know how to squash manually, or rebase, and we use this opportunity to help them learn.  Some already know how to rebase or squash from the CLI, perhaps you are one of those.

Thanks for all your work lately!  We appreciate your contributions.

**Comment by laurmurclar** [comment]
Oh that's a good idea. I do, yes, but I'm not offended! 😄 

No problem 😊

**Comment by rchavarria** [comment]
Yep, I know. I personally don't like how git history is left after using the GitHub button. It's like this:

image

(big black dot)

While I prefer having a git history like this:

image

(blue and purple lines)

Blue and purple lines are got when "Create a merge commit" option is used when merging a PR. It's just a preference. Code doesn't change at all. And it's a *soft rule*, something we can break.

But if you ask "What do you want?, what should PR include?", I would say: "One commit please". I hope I convince you to start loving git histories.
